models/property.model.js

```diff
@@ -52,7 +52,7 @@ const getFilteredProperty = (p) => {
     let query = {
         sql: `SELECT p.*, i.imagePath FROM property as p
               NATURAL JOIN property_photos as i
-              WHERE i.photoId = 1 `
+              WHERE i.photoId > 0 `
```

Comment:
why

Reply:
I don't understand the condition of photoId=1. I felt it is unnecessary (over constrained).

Reply:
So same thing also applies here. I'll also revert it back